tee-worker/ts-tests/identity.test.ts

```diff
@@ -247,26 +321,42 @@ describeLitentry('Test Identity', (context) => {
         //Bob verify extension substrate identities
         const [substrate_extension_identity_verified] = (await verifyIdentities(
             context,
-            context.defaultSigner[1],
+            context.substrateWallet.bob,
             aesKey,
             true,
             [substrateExtensionIdentity],
             [substrateExtensionValidationData]
         )) as IdentityGenericEvent[];
 
         //Alice
-        assertIdentityVerified(context.defaultSigner[0], twitter_identity_verified);
-        assertIdentityVerified(context.defaultSigner[0], ethereum_identity_verified);
-        assertIdentityVerified(context.defaultSigner[0], substrate_identity_verified);
+        assertIdentityVerified(context.substrateWallet.alice, twitter_identity_verified);
+        assertIdentityVerified(context.substrateWallet.alice, ethereum_identity_verified);
+        assertIdentityVerified(context.substrateWallet.alice, substrate_identity_verified);
         //Bob
-        assertIdentityVerified(context.defaultSigner[1], substrate_extension_identity_verified);
+        assertIdentityVerified(context.substrateWallet.bob, substrate_extension_identity_verified);
     });
+    step('check IDGraph after createIdentity', async function () {
```

Comment:
We also need to check the challenge code is removed after the identity is verified :)